Engine/otoolstuff.cpp

```diff
@@ -77,83 +85,80 @@ void _moduleLibraryFixer(const std::string & moduleLibraryPath, bool engineCall,
 			if(printStuff)
 				std::cout << "- Now checking and fixing otool paths for file '" << path.string() << "'.\n";
 	
-			std::string libDir		= stringUtils::replaceBy(path.string(), " ", "\\ "),
-						otoolCmd	= "otool -L " + libDir,
+			std::string libPath		= stringUtils::replaceBy(path.string(), " ", "\\ "),
+						otoolCmd	= "otool -L " + libPath,
 						otoolOut	= _system(otoolCmd);
 			auto		otoolLines	= stringUtils::splitString(otoolOut, '\n');
+
+			std::string libName 	= path.stem().string() + path.extension().string();
+
+			std::cout << "libPath: " << libPath << std::endl;
+			std::cout << "libName: " << libName << std::endl;
 	
-			/*if(printStuff)
-			{
-				std::cout << "- jaspRCPP_postProcessLocalPackageInstall used otool -L on " << libDir;
-				std::cout << " and found this output:\n";
-		
-				for(const auto & line : otoolLines)
-					std::cout << line << std::endl;
-			
-			}*/
-			
-			//ok otoolLines[1] represents the "id" of the lib but we do not need to change it because it probably points directly back to itself. The other lines however we should change
-	
-			for(size_t i=2; i<otoolLines.size(); i++)
+			for(size_t i = 1; i < otoolLines.size(); i++)
```

Comment:
I think, not 100% sure, that I start from 1 because of these guys, they are completely broken, and either their `id` includes the `@rpath` or its one of their entities, and I didn't want to miss them. 

```
"@rpath/libtbbmalloc.dylib"
"@rpath/libtbbmalloc_proxy.dylib"
"@rpath/libtbb.dylib"
```